db/migrate/20160112092557_add_valid_dates_to_business_entity.rb

```diff
@@ -0,0 +1,16 @@
+class AddValidDatesToBusinessEntity < ActiveRecord::Migration
+  def up
+    add_column :business_entities, :valid_from, :datetime
+    add_column :business_entities, :valid_to, :datetime
+    remove_index :business_entities, name: :unique_office_jurisdiction
+    add_index :business_entities, [:office_id, :jurisdiction_id, :valid_to], name: :unique_active_office_jurisdiction
+    execute('UPDATE business_entities SET valid_from = created_at')
```

Comment:
I think we'll have to set the `valid_from` to an older date, won't we? Something like the oldest application creation date. Because if we want to generate the financial report, it will only work on date ranges of valid business entities and therefore it would ignore some. Or won't it?